pennylane/ops/qubit.py

```diff
@@ -559,6 +694,15 @@ class BasisState(Operation):
     par_domain = "A"
     grad_method = None
 
+    @staticmethod
+    def decomposition(n, wires=None):
```

Comment:
That should then use the `BasisStatePreparation` template once it is finally merged.

Reply:
Yep!

Reply:
We should do also add it for `QubitStateVector` -> Mottonen :)